apps/aecore/test/aecore_forking_SUITE.erl

```diff
@@ -0,0 +1,432 @@
+     || E <- [block_created, chain_sync, app_started]].
+
+events_since(N, EvType, TS) ->
+    call_proxy(N, {events, EvType, TS}).
+
+subscribe(N, Event) ->
+    call_proxy(N, {subscribe, Event}).
+
+unsubscribe(N, Event) ->
+    call_proxy(N, {unsubscribe, Event}).
+
+-define(PROXY_CALL_RETRIES, 5).
+
+call_proxy(N, Req) ->
+    call_proxy(N, Req, ?PROXY_CALL_RETRIES, 3000).
+
+call_proxy(N, Req, Tries, Timeout) when Tries > 0 ->
+    Ref = erlang:monitor(process, {?PROXY, N}),
+    {?PROXY, N} ! {self(), Ref, Req},
+    receive
+        {'DOWN', Ref, _, _, noproc} ->
+            ct:log("proxy not yet there, retrying in 1 sec...", []),
+            receive
+            after 1000 ->
+                    call_proxy(N, Req, Tries-1, Timeout)
+            end;
+        {'DOWN', Ref, _, _, Reason} ->
+            error({proxy_died, N, Reason});
+        {Ref, Result} ->
+            erlang:demonitor(Ref),
+            Result
+    after Timeout ->
+            error(proxy_call_timeout)
+    end;
+call_proxy(N, _, _, _) ->
+    erlang:error({proxy_not_running, N}).
+
+
+
+%% ==================================================
+%% Private functions
+%% ==================================================
+
+
+stop_node(N, Config) ->
+    cmd(["(cd ", node_shortcut(N, Config),
+         " && ./bin/epoch stop)"]).
+
+%% Split the DataDir path at "_build"
+top_dir(DataDir) ->
+    [Top, _] = re:split(DataDir, "_build", [{return, list}]),
+    Top.
+
+make_multi(Config) ->
+    Top = ?config(top_dir, Config),
+    ct:log("Top = ~p", [Top]),
+    Epoch = filename:join(Top, "_build/test/rel/epoch"),
+    [setup_node(N, Top, Epoch, Config) || N <- [dev1, dev2, dev3]].
+
+setup_node(N, Top, Epoch, Config) ->
+    ct:log("setup_node(~p,Config)", [N]),
+    DDir = node_shortcut(N, Config),
+    filelib:ensure_dir(filename:join(DDir, "foo")),
+    cp_dir(filename:join(Epoch, "releases"), DDir ++ "/"),
+    cp_dir(filename:join(Epoch, "bin"), DDir ++ "/"),
+    symlink(filename:join(Epoch, "lib"), filename:join(DDir, "lib")),
+    %%
+    CfgD = filename:join([Top, "config/", N]),
+    RelD = filename:dirname(
+             hd(filelib:wildcard(
+                    filename:join(DDir, "releases/*/epoch.rel")))),
+    cp_file(filename:join(CfgD, "sys.config"),
+            filename:join(RelD, "sys.config")),
+    cp_file(filename:join(CfgD, "vm.args"),
+            filename:join(RelD, "vm.args")),
+    delete_file(filename:join(RelD, "vm.args.orig")),
+    delete_file(filename:join(RelD, "sys.config.orig")),
+    TestD = filename:join(filename:dirname(code:which(?MODULE)), "data"),
+    cp_file(filename:join(TestD, "sync_SUITE.config"),
+            filename:join(DDir , "sync_SUITE.config")).
+
+
+
+cp_dir(From, To) ->
+    cmd(["cp -r ", From, " ", To]).
+
+cp_file(From, To) ->
+    {ok, _} = file:copy(From, To),
+    ct:log("Copied ~s to ~s", [From, To]),
+    ok.
+
+symlink(From, To) ->
+    ok = file:make_symlink(From, To),
```

Comment:
This seems to conflict sync_suite and this suite. Re-suing same link>

Reply:
I get:
```
%%% aecore_sync_SUITE ==> init_per_suite: FAILED
%%% aecore_sync_SUITE ==> {{badmatch,{error,eexist}},
 [{aecore_sync_SUITE,symlink,2,
                     [{file,"/Users/luca/dev/aeternity/epoch/_build/test/lib/aecore/test/aecore_sync_SUITE.erl"},
                      {line,574}]},
  {aecore_sync_SUITE,setup_node,4,
                     [{file,"/Users/luca/dev/aeternity/epoch/_build/test/lib/aecore/test/aecore_sync_SUITE.erl"},
                      {line,542}]},
  {aecore_sync_SUITE,'-make_multi/1-lc$^0/1-0-',4,
                     [{file,"/Users/luca/dev/aeternity/epoch/_build/test/lib/aecore/test/aecore_sync_SUITE.erl"},
                      {line,534}]},
  {aecore_sync_SUITE,init_per_suite,1,
                     [{file,"/Users/luca/dev/aeternity/epoch/_build/test/lib/aecore/test/aecore_sync_SUITE.erl"},
                      {line,80}]},
  {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1539}]},
  {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1140}]},
  {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,987}]}]}
%%% aecore_sync_SUITE ==> start_first_node (group two_nodes): SKIPPED
```

Cc @uwiger

Reply:
What do you mean?

Reply:
They should definitely be different link names

Reply:
The symlink is placed under `_build/test/logs/` to reduce the path names in the start and stop commands, but also to make it easier to navigate to the logs during debugging. It just needs to be a different symlink name for different suites. Alternatively, make sure to delete any existing symlink first (though that might confuse the poor developer trying to debug the sync_SUITE)